### ISSUE TITLE
Extremely slow discovery

### ISSUE THREAD

**Issue by NNEU-1** [katoemba/SwiftUPnP#1]
Hi, 

Thanks for sharing this library.
I found that discovery is extremely slow. In my testing, it takes several minutes for the first device to be discovered. 

Discovery in UPnP Atom was also not fast, but still much faster than this.
Would be nice if it could be fixed in the future :)

**Comment by katoemba** [comment]
I have seen discovery issues with BubbleUPnP running in docker, in combination with the iOS simulator. On macOS and a real device discovery of AssetUPnP, BubbleUPnP and upmpdcli is all working well (<3 seconds).

Archiving device properties, so they can be recreated immediately on startup, is not yet implemented.

**Comment by TijenderSingh** [comment]
@katoemba  do we need to enable multicast for debugging as well? or is there any workaround to test it with real device, as I am not getting any device list on simulator.
i want to discover devices like, apple tv, samsung , fire, googel chome cast etc.

**Comment by lmcd** [comment]
Yeah I'm seeing 5-10 minutes for the first discovery to arrive, and sometimes they don't arrive at all.
This is on macOS on a real network with real UPnP devices
